flit-client/src/components/ConversationPage/index.tsx

```diff
@@ -34,6 +86,12 @@ timedLines.push({
   timestamp: new Date('2020-02-20T12:34Z'),
 });
 
+// add any autogenerated fields and process into final format
+const finalLines = timedLines
+  .map((line) => ({...line, uid: generateId(line.name)}))
+  .map((line) => ({...line, avatar: generateAvatar(line)}))
+  .map((line) => ({user: extractUser(line), msg: extractMessage(line), msgStatus: generateStatus(line)}))
```

Comment:
The amount of this file that's generating mock data is starting to overwhelm the amount of it that's real. Probably should extract the mock logic to its own file soon so we can more clearly separate it from the actual code.